console/.env.example

```diff
@@ -0,0 +1,8 @@
+PG_CONSOLE_DOMAIN=
+
+PG_CONSOLE_API_URL=$PG_CONSOLE_DOMAIN/api/v1
+PG_CONSOLE_AUTH_TOKEN=
+PG_CONSOLE_DOCKER_IMAGE=autonbase/automation:latest
```

Comment:
This `PG_CONSOLE_DOCKER_IMAGE` variable is optional since the Console API service will automatically launch the automation image that matches the console version (see API service [configuration](https://github.com/vitabaks/autobase/blob/master/console/service/README.md#configuration)).

Using the `latest` versions is great for testing but not recommended for production installations. I think keeping the ‘latest’ tag is fine, as it allows testing the most recent changes, but it might be worth adding a note to clarify this.

Reply:
Make sense. I was sharing the configuration that I got working, but many more tweaks can be done, like exposing all environment variables depending on the project.

Reply:
I get you, I can do it on my own.